fathomnet/scripts/fathomnet_generate.py

```diff
@@ -182,7 +202,7 @@ def generate_coco_dataset(ims: List[AImageDTO], output_dir: str) -> bool:
             id=image_id,
             width=image.width,
             height=image.height,
-            file_name=image.url.split('/')[-1],
+            file_name=f"{image.uuid}.{image.url.split('.')[-1]}",
```

Comment:
Totally. We should keep that in mind going forward

On a related note, I did not update the file_name for the VOC generator. It looks like it might already do: line 154 uses UUID to identify the xml annotations. Likewise in `write_voc()` at line 57-60 it appears to using the UUID implicitly in `image.to_pascal_voc`.